### ISSUE TITLE
Service Message Popups

### ISSUE THREAD

**Issue by L0ric0** [cylgom/ly#38]
If tlp or plymouth are not installed Service messages will still popup over ly.

I will look for a solution myself but as I don't have any experience in c it could take time.

**Comment by cylgom** [comment]
Actually this can get really tricky, we had a hard time with Oxodao trying to fix this, and the only solution we could find is described in the readme. Another working option could be to edit you kernel parameters.
Can you tell me if the first method works for you?

**Comment by L0ric0** [comment]
The method described in the readme doesn't work for me or I would have to specify each of these services. 

As for the kernel parameters, I can suppress them withe the quiet option, but this also suppresses errors and I haven't found one that lets me redirect messages to a specific tty or file.

Is it possible to catch all input to the tty and redirect it to another?
I will see if I can find stuff in the agetty source as it and other programs like it have to have methods to only let errors show errors.

**Comment by cylgom** [comment]
@RobertZenz this easy to do, but not quite a clean approach.
@L0ric0 you have to specify the last service launching on your computer that gives error messages.

**Comment by RobertZenz** [comment]
No, it isn't, but it would be a quick and easy workaround. Especially when made optional.

**Comment by L0ric0** [comment]
@cylgom Yes I know but systemd works asynchronous where it can and so I have multiple services that can be the last to launch. But I have found that you can define a target in the After section of a service. So adding:

After=multi-user.target 

to the service file will launch ly only when all services of multi-user.target are started circumventing the issue of knowing the last service to start.
